{"name": "Проверяем метод remove_ingredient() - удаление ингредиента: неправильный индекс", "status": "broken", "statusDetails": {"message": "IndexError: list assignment index out of range", "trace": "self = <test_burger.TestBurger object at 0x000001C77B982890>, setup_sauce = <Mock id='1956284506896'>\nsetup_filling = <Mock id='1956284504848'>\n\n    @allure.title('Проверяем метод remove_ingredient() - удаление ингредиента: неправильный индекс')\n    def test_remove_ingredient_invalid_index_no_changes(self, setup_sauce, setup_filling):\n        # создаем бургер и 2 ингредиента и добавляем их (с индексами 0 и 1)\n        burger = Burger()\n        mock_sauce = setup_sauce\n        mock_filling = setup_filling\n        burger.add_ingredient(mock_sauce)\n        burger.add_ingredient(mock_filling)\n        # пытаемся удалить ингредиент с индексом 2\n>       burger.remove_ingredient(2)\n\ntests\\test_burger.py:145: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <burger.Burger object at 0x000001C77BA4A990>, index = 2\n\n    def remove_ingredient(self, index: int):\n>       del self.ingredients[index]\nE       IndexError: list assignment index out of range\n\nburger.py:26: IndexError"}, "start": 1702833758722, "stop": 1702833758722, "uuid": "2ee0f688-71ef-4b29-a9bb-911731226f33", "historyId": "471c6da4a1ac45e23bed04c5ee4e8528", "testCaseId": "471c6da4a1ac45e23bed04c5ee4e8528", "fullName": "tests.test_burger.TestBurger#test_remove_ingredient_invalid_index_no_changes", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_burger"}, {"name": "subSuite", "value": "TestBurger"}, {"name": "host", "value": "DESKTOP-O6MNHQN"}, {"name": "thread", "value": "14256-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_burger"}]}